8.4/js/firstjs.js

```diff
@@ -0,0 +1,46 @@
+function getTriangleArea (a, h) {
+	
+	if (a > 0 && h > 0) {
+		console.log('Podane wartości są prawidłowe');
+		var value = (a * h) / 2;
+	} else {
+		console.log('Podane wartości są nieprawidłowe');
+	}
+		return value;
+}
+
+var getTriangle2Area = function (a, h) {
+
+	if (a > 0 && h > 0) {
+		console.log('Podane wartości są prawidłowe');
+		var value = (a * h) / 2;
+	} else {
+		console.log('Podane wartości są nieprawidłowe');
+	}
+		return value;
+};
+
+var Triangle3 = function getTriangle3Area (a, h) {
+
+	if (a > 0 && h > 0) {
+		console.log('Podane wartości są prawidłowe');
+		var value = (a * h) / 2;
+	} else {
```

Comment:
wtedy ten `else` jest niepotrzebny tylko od razu `console.log()`, ze jest blad, bo jak nie ma bledu to wyżej `return` wyjdzie odpowiednio wcześnie z funkcji. Mamy więc pewność, że jak kod doszedł do tego miejsca to jest błąd do wyświetlenia.